src/lib/Collapsible.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import { IconChevronRight, IconChevronDown } from "./Icons";
+
+export default function Collapsible({
+  title,
+  children,
+}: {
+  title: string;
+  children: React.ReactNode;
+}) {
+  const [collapsed, setCollapsed] = React.useState(false);
+
+  return (
```

Comment:
Nitpicking here, but did you consider https://www.radix-ui.com/docs/primitives/components/collapsible ? It would cover all of the [acceptability concern here ](https://www.w3.org/TR/wai-aria-practices-1.2/#disclosure)

Reply:
Didn't really look into the application from an accessibility perspective. Found some more a11y problems. Will add a new GitHub issue to address this. Will update this comment with issue# in a while.

Reply:
Sounds good to me !

Reply:
Captured this concern in https://github.com/hasura/public-graphiql/issues/2